### DISCUSSION TITLE
Request ICMP module

### DISCUSSION THREAD

**Discussion by iliaselmatani** [thinkst/opencanary#275] [Feature Requests]
Missing support for ICMP.
Would be nice to have ICMP as module.
This would be a very useful detection besides the module portscan.

**Comment by ghost** [comment]
I would second that. As OpenCanary project, ICMP is surely needed.

**Comment by folkeorg** [comment]
ICMP goes undetected now i.e. ping-scan of devices is posssible so this is a needed feature.

**Comment by jayjb** [comment]
Thanks for reporting this folks. I'll bring this up with the team and we can discuss it.

**Comment by Blu3panda-dev** [comment]
Is there more news about the ICMP feature?
